app/src/App.tsx

```diff
@@ -53,68 +51,83 @@ export default () => {
 						gridTemplateRows: '60px calc(50% - 60px) 60px calc(50% - 60px)'
 					}}
 				>
-					<div style={{ border: '1px solid blue', gridArea: '1 / 1 / span 1 / span 3' }}>
+					<NeonBox style={{ gridArea: '1 / 1 / span 1 / span 3' }}>
 						<Title />
-					</div>
+					</NeonBox>
 					<StatsWrapper>
 						<div style={{ height: '100%', minHeight: '300px', position: 'relative' }}>
 							<Stats />
 							<Saves />
 						</div>
 					</StatsWrapper>
-					<div style={{ border: '1px solid purple', gridArea: '2 / 2 / span 1 / span 1' }}>
+					<NeonBox style={{ gridArea: '2 / 2 / span 1 / span 1' }}>
 						<Actions />
-					</div>
-					<div style={{ border: '1px solid red', gridArea: '3 / 1 / span 1 / span 2' }}>
+					</NeonBox>
+					<NeonBox style={{ gridArea: '3 / 1 / span 1 / span 2' }}>
 						<HealthBar />
-					</div>
-					<InventoryWrapper>
-						<Inventory />
-					</InventoryWrapper>
-					<CyberneticWrapper>
-						<Cybernetics />
-					</CyberneticWrapper>
-					<SkillsWrapper style={{ border: '1px solid blue', gridArea: '2 / 3 / span 3 / span 1' }}>
-						<Skills />
-					</SkillsWrapper>
+					</NeonBox>
+					<InventoryWrapper component={Inventory} />
+					<CyberneticWrapper component={Cybernetics} />
+					<SkillsWrapper component={Skills} />
 				</div>
 			</characterContext.Provider>
 		</>
 	);
 };
 
-const StatsWrapper = styled.div`
+const NeonBoxStyle = css`
+	margin: -1px;
+	padding: 4px;
+	border: 1px solid #9fd2ff;
+	box-shadow: 0 0 1px #9fd2ff, inset 0 0 1px #9fd2ff, 0 0 5px #08f, inset 0 0 5px #08f, 0 0 8px #08f, inset 0 0 8px #08f;
+`;
+
+const NeonBox = styled.div`
+	${NeonBoxStyle};
+`;
+
+interface WrapperProps {
+	className?: string;
+	component: () => JSX.Element;
+}
+
+const GenericWrapper = ({ className, component: Component }: WrapperProps) => (
+	<div className={className}>
+		<Component />
+	</div>
+);
+
+const StatsWrapper = styled(NeonBox)`
 	overflow: scroll;
-	border: 2px solid #00ffff;
 	gridarea: 2 / 1 / span 1 / span 1;
 	overflow-y: scroll;
 	&::-webkit-scrollbar {
 		display: none;
 	}
 `;
 
-const CyberneticWrapper = styled.div`
+const CyberneticWrapper = styled(GenericWrapper)`
+	${NeonBoxStyle};
 	overflow-y: scroll;
-	border: 1px solid blue;
-	gridarea: 4 / 2 / span 1 / span 1;
+	grid-area: 4 / 2 / span 1 / span 1;
 	&::-webkit-scrollbar {
 		display: none;
 	}
 `;
 
-const SkillsWrapper = styled.div`
-	/* border: 1px solid blue;
-	gridarea: 2 / 3 / span 3 / span 1; this is the same as the inline style, but doesn't work */
+const SkillsWrapper = styled(GenericWrapper)`
+	${NeonBoxStyle};
```

Comment:
Never mind, I see now. However, I do think that generic styles like that that will be used elsewhere should go in a file and exported. That doesn't look like it's happening now though, so that's fine.